eslint-plugin-skyscanner-no-axios/src/rules/no-axios.test.js

```diff
@@ -0,0 +1,55 @@
+const { RuleTester } = require('eslint');
+
+const noAxios = require('./no-axios');
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+});
+
+ruleTester.run('no-moment', noAxios, {
```

Comment:
haha let's change to `no-axios`, otherwise all looks good!

Reply:
good catch!

Reply:
fixed